src/client/clientmap.cpp

```diff
@@ -582,6 +595,9 @@ void ClientMap::updateDrawList()
 
 void ClientMap::touchMapBlocks()
 {
+	if (!m_loops_occlusion_culler)
```

Comment:
Condition inverted?
Also since the loops culler does not reset the blocks' usage times, this still needs to be called anyway, right?